src/presentation/react/core/components/sync-wizard/common/SummaryStep.tsx

```diff
@@ -199,7 +204,11 @@ const SaveStep = ({ syncRule, onCancel }: SyncWizardStepProps) => {
                     >
                         <ul>
                             {syncRule.excludedIds.map(id => {
-                                const element = _(metadata).values().flatten().find({ id });
+                                const values = Object.keys(metadata).map(
+                                    key => metadata[key as keyof MetadataEntities]
+                                );
+
+                                const element = values.flat().find(element => element?.id === id);
```

Comment:
This breaks in old browsers (old meaning some for example mobile are this month old)